cves/CVE-2011-4136.yml

```diff
@@ -267,7 +300,7 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 20
```

Comment:
Its more than just user input validation. Its about securing the session so an attacker cannot manipulate it